### PR TITLE
revise documentation for YellowBox API

### PR DESCRIPTION
**Pull request by sgny**
@cknitt already included an example and the documentation was quite complete, but I felt it might be helpful to flesh out the documentation a bit further.

### CODE REVIEW

File: reason-react-native/src/apis/YellowBox.md [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
 title: YellowBox
-wip: true
+officialDoc: https://facebook.github.io/react-native/docs/debugging#warnings
 ---
 
-```reason
-[@bs.module "react-native"] [@bs.scope "YellowBox"]
-external ignoreWarnings: array(string) => unit = "";
+Warning messages are displayed in white on a yellow background; which is known
+as a YellowBox. Warnings may be trigged with the `warn` method already included
```
Comment (MoOx) [inline reason-react-native/src/apis/YellowBox.md]: I would go for ``Warnings may be trigged with the `Js.Console.warn` method included BuckleScript`` (so you can directly copy/paste/visualise the call to make

Comment (cknitt) [reply reason-react-native/src/apis/YellowBox.md]: I would write ```Warnings may be trigged with the `Js.Console.warn` function included in BuckleScript.```

File: reason-react-native/src/apis/YellowBox.md [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
+the BuckleScript module `Js.Console`. This module relates to the `YellowBox`
+module in React Native.
```
Comment (cknitt) [inline reason-react-native/src/apis/YellowBox.md]: I'd remove the last sentence.

**Comment by MoOx** [comment]
@sgny one nice little thing is that we should prefix commit with module name when possible to help for changelog (here adding `reason-react-native: ` would have been nice ^^).
We can do it when using `Squash & Merge` has it allow you to adjust the commit message.

**Pull request merged**